src/components/Inventory/withdraw/BatchWithdrawFlow.tsx

```diff
@@ -19,7 +21,13 @@ type BatchWithdrawFlowProps = {
 export default function BatchWithdrawFlow(props: BatchWithdrawFlowProps): JSX.Element {
   const { organization, batchIds } = props;
   const [flowState, setFlowState] = useState<FlowStates>('purpose');
-  const [records, setRecords] = useForm<NurseryWithdrawal[]>([]);
+  const [record, setRecord] = useForm<NurseryWithdrawal>({
```

Comment:
this is the source withdrawal object, it's a single withdrawal with a bulk of batches. we got it wrong earlier, i messed up a bit. the array is only on the batchWithdrawals property which contains batch specific info.

Reply:
Excellent!